src/views/virtualmachines/details/tabs/network/copmonents/modal/utils/helpers.ts

```diff
@@ -0,0 +1,16 @@
+import { adjectives, animals, uniqueNamesGenerator } from 'unique-names-generator';
+
+import { V1VirtualMachine } from '@kubevirt-ui/kubevirt-api/kubevirt';
+
+export const generateNicName = () => {
+  return `nic-${uniqueNamesGenerator({
+    dictionaries: [adjectives, animals],
+    separator: '-',
+  })}`;
+};
+
+export const isPodNetworkExists = (vm: V1VirtualMachine): boolean =>
```

Comment:
This function name and `isNetworkNameStartWithPod` both read strangely to me. I think `podNetworkExists` and `networkNameStartsWithPod` sound more natural.

Reply:
changed thanks :)